e2e-tests/portal/cypress/e2e/journeys/maker-about-supply-contract/fill-in-about-buyer-page-happy-path.spec.js

```diff
@@ -1,29 +1,20 @@
-const { contract, contractAboutSupplier, contractAboutBuyer, contractAboutPreview, defaults } = require('../../pages');
+const { contract, contractAboutSupplier, contractAboutBuyer, contractAboutFinancial } = require('../../pages');
 const partials = require('../../partials');
 const MOCK_USERS = require('../../../../../e2e-fixtures');
-const aDealWithAboutSupplyContractComplete = require('./dealWithFirstPageComplete.json');
 
 const { BANK1_MAKER1 } = MOCK_USERS;
 
 context('about-supply-contract', () => {
-  let deal;
-
   before(() => {
-    cy.insertOneDeal(aDealWithAboutSupplyContractComplete, BANK1_MAKER1).then((insertedDeal) => {
-      deal = insertedDeal;
-    });
+    cy.createBssEwcsDeal({});
   });
 
   it('A maker picks up a deal with the supplier details completed, and fills in the about-buyer-contract section, using the companies house search.', () => {
```

Comment:
Is the supplier details complete at this stage - I don't believe so - if so, we can simplify this test description.

Reply:
Simplified the test description